requirements-dev.txt

```diff
@@ -30,3 +30,4 @@ Sphinx==3.3.0
 recommonmark==0.6.0
 sphinx-autodoc-typehints==1.11.1
 sphinx-rtd-theme==0.5.0
+vtk
```

Comment:
We should really avoid adding large dependencies like this if it can be avoided.

Reply:
This is for the inverse of non-rigid transformations. I wrote them in both vtk and simple itk, so perhaps we could convert the simple itk code to itk and avoid another import? I think I'd need some help though

Reply:
Other options are simply not providing the inverse for non-rigid transformations (torchvision does this) or not adding it as a dependency (an exception is raised if the user tries to do the inverse but vtk/simpleitk is missing) and allowing the user to download it if desired.

Reply:
I think adding it as an optional dependency with `optional_import` is the way to do that at least for now.

Reply:
I only ever import vtk with `optional_import`, but I think that's true of everything in `requirements-dev.txt`.